docs/build-script/examples/change-docroot/index.md

```diff
@@ -8,15 +8,8 @@ Tugboat looks.
 
 ## apache
 
-```
-tugboat-init:
-    ln -sf ${TUGBOAT_ROOT}/public_html /var/www/html
-```
+[import:1-3, lang:"makefile", template:"ace"](Makefile)
```

Comment:
This is awesome! just include specific lines? I think we found a winner with this plugin!